### PR TITLE
move Facility locator back to v0/ path, replicate vbeta/ temporarily via routes

### PR DESCRIPTION
**Pull request by dhall**
Once we are comfortable with the new vbeta controller, this is the PR to swap the names and pull the old code.

### CODE REVIEW

**Comment by dhall** [comment]
I believe it to be complete as is.. with the following approach (just slightly modified from what you said):
1. v0/ replaces vbeta/ (and routing entry redirect vbeta to v0, essentially duplicating)
2. old functionality removed

After merge and verify then:
1. vets-website -- switch facility locator from vbeta/ to v0/
2. vets-ap -- remove vbeta/ route

**Comment by saneshark** [comment]
That could work, but will want to test that the redirects work correctly as vets-website might not automatically follow redirects.

**Comment by dhall** [comment]
@saneshark I just tested and verified locally -- to clarify, these aren't 301's, it's rails internal routing here are the lines:

`get '/vbeta/facilities/va', to: 'v0/facilities/va#index'`
`get '/vbeta/facilities/va/:id', to: 'v0/facilities/va#show'`

I'm continuing (for now) to advertise the route but changed the controller that handles the request

File: app/models/base_facility.rb [hunk 1]
```diff
@@ -81,9 +81,10 @@ def find_facility_by_id(id)
     # FIXME: using common collection here to have access to #metadata for serializer output but long term we should
     # refactor and mix that in -- https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9376
     def query(params)
+      return common_collection([]) unless params[:bbox]
```
Comment (saneshark) [inline app/models/base_facility.rb]: I'm afraid this common collection won't have the correct type. Might run into javascript parsing issues without or schema failures. Do you have a spec for this scenario?

Comment (dhall) [reply app/models/base_facility.rb]: This was added to replicate original behavior and make the swagger specs pass. In practice the bbox param is required. I'm up for changing this behavior in a future PR but felt like we should make the switchover with matching functionality.

Comment (saneshark) [reply app/models/base_facility.rb]: probably ok if type is set in the serializer.

**Pull request merged**